app/views/schemes/show.html.erb

```diff
@@ -10,19 +10,28 @@
 
 <%= render partial: "organisations/headings", locals: { main: @scheme.service_name, sub: nil } %>
 
-<%= render SubNavigationComponent.new(items: scheme_items(request.path, @scheme.id, "Locations")) %>
+<% if FeatureToggle.new_locations_table_layout_enabled? %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
```

Comment:
Also, why are you changing the schemes/show page, I can't see any changes to this page in the designs for the ticket?